### PR TITLE
Adds a follow button for all gangsters in check antag button

### PR DESCRIPTION
**Pull request by lzimann**
It kinda bothers me that only gang leaders get that

### CODE REVIEW

File: code/modules/admin/player_panel.dm [hunk 1]
```diff
@@ -414,6 +414,7 @@
 				if(M)
 					dat += "<tr><td><a href='?_src_=holder;adminplayeropts=\ref[M]'>[M.real_name]</a>[M.client ? "" : " <i>(ghost)</i>"][M.stat == 2 ? " <b><font color=red>(DEAD)</font></b>" : ""]</td>"
 					dat += "<td><A href='?priv_msg=[M.ckey]'>PM</A></td></tr>"
+					dat += "<td><A href='?_src_=holder;adminplayerobservefollow=\ref[M]'>FLW</a></td></tr>"
```
Comment (optimumtact) [inline code/modules/admin/player_panel.dm]: Pls to be using the macro
https://github.com/tgstation/tgstation/pull/19070/files#diff-a51fd60b9f94372d01da166ee2cb3401R41

Comment (lzimann) [inline code/modules/admin/player_panel.dm]: can you merge this and then i change everything later?

Comment (optimumtact) [inline code/modules/admin/player_panel.dm]: No

**Comment by optimumtact** [comment]
Please do the needful

**Comment by lzimann** [comment]
gonna make another pr later and replace them all

**Comment by optimumtact** [comment]
All of them!~

**Comment by optimumtact** [comment]
Madman

**Pull request closed without merging**